docs/hardware/devices/lilygo/index.mdx

```diff
@@ -40,11 +40,12 @@ Inexpensive basic ESP32-based boards.
 
 ## [T-Deck](./tdeck/)
 
-Standalone device with screen and keyboard
+Standalone device with screen and keyboard. T-Deck Plus also comes with a 2000 mAh battery. Both includes a secondary MCU that manages keyboard: ESP32-C3
```

Comment:
This level of detail is probably better kept inside the tdeck documentation rather than on the index page here

Reply:
I's not the index page but only lilygo section.
But I updated my branch taking note to your comment.